account/accounts_test.go

```diff
@@ -412,3 +415,17 @@ func (s *ManagerTestSuite) TestAddressToDecryptedAccount() {
 		})
 	}
 }
+
+func (s *ManagerTestSuite) TestCreateAddress() {
+	addr, pub, priv, err := s.accManager.CreateAddress()
+	s.Equal(true, gethcommon.IsHexAddress(addr))
+	s.NoError(err)
+
+	privECDSA, err := crypto.HexToECDSA(priv[2:])
+	s.NoError(err)
+
+	pubECDSA := privECDSA.Public().(*ecdsa.PublicKey)
+	pubStr := hexutil.Encode(crypto.FromECDSAPub(pubECDSA))
+
+	s.Equal(pub, pubStr)
```

Comment:
I guess it should be `s.Equal(pubStr, pub)`. I would recommend to call the arguments `s.Equal(expectedPubKeyStr, pub)`.